### DISCUSSION TITLE
Partner and Multiple tenants

### DISCUSSION THREAD

**Discussion by DanPhoaser** [EvotecIT/O365Essentials#21] [Q&A]
Hey great module! Thanks for putting this together.
Went through the connect-o365admin function but wasn't able to fully determine if this supports interacting with partner tenants using this and if it supports connecting with app registration and client secret/certificate.

**Comment by PrzemyslawKlys** [comment]
It doesn't support any of that at current state. It uses the standard way to connect to tenants like you would usually do when you go to portal.office.com and just log in. No client secrets/no certs. 

I haven't had time to research this at all :( At some point I was able to get it up and running with Device prompt query, but nothing like you mentioned.
